### ISSUE TITLE
[Effect Suggestion] Spawn Random Plane

### ISSUE THREAD

**Issue by pogue** [gta-chaos-mod/ChaosModV#1523]
**Spawn Random Plane**
Not timed
There are many, many cool planes in GTA online that could be spawned at random locations

**Comment by Last0xygen** [comment]
already exists with "Spawn Random Vehicle" :)

**Comment by pogue** [comment]
Again, my suggestion is to have a separate option to "Spawn Random Plane" instead of "Spawn Random Vehicle".

**Comment by kolyaventuri** [comment]
Idea: separate out individual types? "Spawn Random Car", "Spawn Random Plane", etc. vs a singular "Random Vehicle" ?

**Comment by TheRealAyCe** [comment]
Yes, and also introduce groups for that, so that "Spawn random Car" and "Spawn Adder" are in one group. What do you think?

**Comment by pogue** [comment]
I don't know about having the option to spawn individual cars, but stuff like "Spawn Plane" "Spawn Super Car", "Spawn Sports Car", "Spawn Boat", etc. would be more appropriate I think.

**Comment by TheRealAyCe** [comment]
I suggested it like that because there are already effects that spawn very specific vehicles.

**Comment by pogue** [comment]
> I suggested it like that because there are already effects that spawn very specific vehicles.

Yeah I agree. I would like to see more variety in car & vehicle types. I made a lot of suggestions, such as spawning arena war vehicles, spawning import/export special vehicles, spawn as an rc bandito/rc tank/drone and etc, but they all got closed and the author said that "Spawn Random Vehicle" will spawn these. Only once have I seen it spawn a GTA online supercar in DarkViperAU's stream, but it's mostly always civilian/ped street cars.

There's also a bug where they seem to spawn under the map.

**Comment by Last0xygen** [comment]
> but they all got closed and the author said that "Spawn Random Vehicle" will spawn these. Only once have I seen it spawn a GTA online supercar in DarkViperAU's stream, but it's mostly always civilian/ped street cars.

no. "spawn random vehicle" spawns a random vehicle. thats it. ANY vehicle that is currenlty in the game. 
If DarkViper doesn't have a mod installed to "unlock" online vehicles in single player -> they propably wont spawn.
if you watch someone like "GTAMen" for example, you'll see online vehicles getting spawned by this.

> There's also a bug where they seem to spawn under the map.

no. "Spawn Random Vehicle" never spawns a vehicle under the map, you're confusing "Set Player Into Random Vehicle (Seat" with spawning a new Vehicle
